EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/GeneralMessageHandler/GeneralMessageHandler.cs

```diff
@@ -18,30 +18,16 @@ public async Task ProcessCommand(string userInput)
         // TODO: Consider the order of this once the scope of overall commands have been established. For efficency.
 
         var commandParams = userInput.Split(' ');
-        var command = commandParams!.First().ToLower();
+        var command = commandParams![0].ToLower();
```

Comment:
is [0] better than first()?
Curious on this change?

Reply:
my compiler was giving me the shits saying this is the more "modern" approach

Reply:
Check other PR for explanation